configfiles/test-small-911.xml

```diff
@@ -37,7 +37,7 @@
       <Connections911Params name="Connections911Params">
         <graphmlFile name="graphmlFile">../configfiles/graphs/small_911.graphml</graphmlFile>
             <!-- Max ConnsPerVertex = 87 RC + 1 PR -->
-        <connsPerVertex name="connsPerVertex">88</connsPerVertex>
+        <connsPerVertex name="maxConnsPerVertex">88</connsPerVertex>
```

Comment:
Change needs to be made to both the tag name and the `name` attribute. Actually, the latter is intended to be used by Workbench in its GUI, so it should really be more user friendly, such as `name="Maximum Connections per Vertex"`

Also, the same change needs to be made to the other config file.

Reply:
I assume that we have to make all those `name` attributes in the config files more user-friendly. I also foresee that these config files will change as I complete the ESCS model implementation.

Reply:
I agree.